include/nbl/system/IApplicationFramework.h

```diff
@@ -20,6 +24,23 @@ class NBL_API IApplicationFramework : public core::IReferenceCounted
             const system::path& _sharedOutputCWD) : 
             localInputCWD(_localInputCWD), localOutputCWD(_localOutputCWD), sharedInputCWD(_sharedInputCWD), sharedOutputCWD(_sharedOutputCWD)
 		{
+#ifdef _NBL_PLATFORM_WINDOWS_
+            HMODULE res = LoadLibraryExA(_NABLA_DLL_NAME_, NULL, LOAD_LIBRARY_SEARCH_APPLICATION_DIR);
+            if (!res)
+            {
+                const std::string nablaOutputBuildPath = std::filesystem::path(std::string(_NABLA_OUTPUT_DIR_) + std::string(_NABLA_DLL_NAME_)).make_preferred().string();
+                res = LoadLibraryExA(nablaOutputBuildPath.c_str(), NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
+            }
+            if (!res)
+            {
+                const std::string nablaInstallPath = std::filesystem::path(std::string(_NABLA_INSTALL_DIR_) + std::string(_NABLA_DLL_NAME_)).make_preferred().string();
```

Comment:
instead of
```cpp
std::filesystem::path(std::string(X) + std::string(Y)).make_preferred().string()
```
do
```cpp
(system::path(X).make_preferred()/Y).string();
```

Reply:
resolved